AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/AWSAuthBaseTest.swift

```
@@ -18,13 +18,17 @@ class AWSAuthBaseTest: XCTestCase {
 
     let amplifyConfigurationFile = "testconfiguration/AWSCognitoAuthPluginIntegrationTests-amplifyconfiguration"
 
+    func initializeAmplifyWithError() throws {
+        let configuration = try TestConfigHelper.retrieveAmplifyConfiguration(
+            forResource: amplifyConfigurationFile)
+        let authPlugin = AWSCognitoAuthPlugin()
+        try Amplify.add(plugin: authPlugin)
+        try Amplify.configure(configuration)
+    }
+
     func initializeAmplify() {
```

Hoping to get rid of this silent failure function in an upcoming PR.